src/emailservice/Dockerfile

```diff
@@ -1,27 +1,52 @@
+FROM python:2.7-alpine3.8 as base
 
+FROM base as builder
 
-# Use the grpc.io provided Python image as the base image
-FROM grpc/python:1.0
+# gRPC deps
+RUN apk add --update --no-cache \
+    gcc \
+    linux-headers \
+    make \
+    musl-dev \
+    python-dev \
+    g++ \
+    cairo-dev \
+    cairo \
+    openssl-dev \ 
+    gobject-introspection-dev
 
-# download the grpc health probe
+ENV GRPC_PYTHON_VERSION 1.15.0
+
+# Install grpcio
+RUN python -m pip install --upgrade pip
+RUN pip install grpcio==${GRPC_PYTHON_VERSION} grpcio-tools==${GRPC_PYTHON_VERSION}
```

Comment:
I recommend we just put these two inside requirements.txt. Not sure why we need to treat them any differently.

Reply:
Can do. I originally left them in there to mimic the gRPC docker image

Reply:
I think we should just move it to requirements.txt.

Also I recommend keeping the original grpc version. The 1.15 has introduced new methods in the health.proto that we haven't implemented (not sure if it's a problem in Python, but it broke the app in Go).

Reply:
Moved to requirements.txt & rolled versions back to 1.12.1